chainerio/__init__.py

```diff
@@ -84,44 +184,98 @@ def create_handler(uri: str) -> IO:
     return handler
 
 
-def isdir(file_path: str) -> bool:
+def isdir(path: str) -> bool:
+    """return whether the given path is a directory
+
+    Works similar to the `os.isdir` and shares the same parameter list.
```

Comment:
"similar" is inappropriate term as this is a document that describes the specification of functionalities. Sentences like this had better been removed.

Reply:
Addressed in e0df1ba